reverse_replication/README.md

```diff
@@ -93,20 +93,38 @@ go run launcher.go -projectId=my-project -dataflowRegion=us-east1 -instanceId=my
+The Dataflow job that reads change streams and writes to PubSub can get stuck in an infinite loop retrying the same set of records in certain scenarios. These scenarios can occur when there are a lot of changestream records to be read in a short interval of time, which can happen in the following situations:
+- There is an unexpected spike in Spanner activity.
+- The pipeline is started with a date in the past (due to issues that required downtime, such as a bug fix).
+Currently, there is no way to revert this within the same job. More details and recovery steps are below.
+
+#### Recovery steps for the infinite loop
 1. The user must track the last handled window that was successfully processed - this can be obtained from the Dataflow logs or by checking the watermark of the job stage.
-2. Specify a different partition metadata database or drop the previous metadata tables  - since watermarks for a given partition are persisted and hence will be read from that point onwards. So the pipeline must begin with a clean partition metadata database.Note that the metadata tables can be dropped by giving DROP TABLE\<table name\> statements in the Cloud UI.
-3. The current pipeline must be updated - the user must specify the start time of change stream query as the last successfully processed timestamp and the end time of the change stream as the time that would result in ~1GB writes. Example, if average record size is 1KB, then 10,00,000 records should be processed in a window and if the TPS during that window was 20K, then the window must end at 50 second. Detailed steps to update the Dataflow job are given here: https://cloud.google.com/dataflow/docs/guides/updating-a-pipeline#gcloud-cli
-4. Once this window is processed, the pipeline must be updated with next set of start and end times, until the pipeline catches up and the finally the end timestamp need not be passed.
+2. Specify a different partition metadata database or drop the previous metadata tables - since watermarks for a given partition are persisted and hence will be read from that point onwards. So the pipeline must begin with a clean partition metadata database.Note that the metadata tables can be dropped by giving DROP TABLE\<table name\> statements in the Cloud UI.
+3. The current pipeline must be updated - the user must specify the start time of the change stream query as the last successfully processed timestamp and the end time of the change stream as the time that would result in ~1GB writes. Example, if the average record size is 1KB, then 10,00,000 records should be processed in a window and if the TPS during that window was 20K, then the window must end at 50 seconds. Detailed steps to update the Dataflow job are given here.
+4. Once this window is processed, the pipeline must be updated with the next set of start and end times, until the pipeline catches up and finally the end timestamp need not be passed.
+
+## Double Load on Spanner
```

Comment:
How is this a limitation of the reverse replication? If anything - this must be filtered from the forward migration job. This is not really a limitation - more like it's a resource impact. This side effect is covered in the user guide.

Reply:
Yes its not exactly a reverse replication limitation. Even though this section is titled limitations, I was thinking of putting in all potential issues and things to look out for in one single place, instead of having some parts of the info in place and some in other. If there a better title suggestion we can look into that.

Reply:
Please remove from here, have added this to user guide.